style.css

```diff
@@ -14,9 +18,11 @@ button{
   background-color: red;
```

Comment:
The best practice for arranging CSS attributes inside of a selector is to put positioning attributes (like margin, text-align, padding, etc.) at the top, with things like border and font styling at the bottom. 

Think BIG > small.